plugin/clang.vim

```diff
@@ -46,6 +46,10 @@ if !exists('g:clang_exec')
   let g:clang_exec = 'clang'
 endif
 
+if !exists('g:gcc_exec')
```

Comment:
better to change to `clang_gcc_exec` ? All global vars should start with `clang_`